source/documents/modify.txt

```diff
@@ -6,16 +6,29 @@ Modify Documents
 
 .. default-domain:: mongodb
 
-.. important::
-
-   Modifying documents is not permitted in
-   :guilabel:`MongoDB Compass Readonly Edition`.
-
 You can edit existing documents in your collection. When you edit
 a document, |compass-short| performs a
 :manual:`findAndModify </reference/method/db.collection.findAndModify/>`
 operation to update the document.
 
+Limitation
```

Comment:
```suggestion
Limitations
```

Reply:
Yes should definitely be `Limitations`, my mistake. Thanks!